src/components/cards.js

```diff
@@ -145,6 +131,23 @@ const RecipeCard = ({
               {time}
             </span>
           </div>
+          <Flex sx={{ flexDirection: `row` }}>
+            <div sx={{ display: `flex`, alignItems: `center`, ml: `3` }}>
+              <UpvoteCardIcon recipeId={id} recipeName={name} />
```

Comment:
Does `UpvoteCardIcon` need `recipeName`?

Reply:
It looks like you just used what was already there but we can probably take it out at some point if it's not getting used anyway

Reply:
Just removed it